usr/sbin/registercloudguest

```diff
@@ -200,6 +200,15 @@ if args.user_smt_ip or args.user_smt_fqdn or args.user_smt_fp:
         print(msg, file=sys.stderr)
         sys.exit(1)
 
+    if not utils.instance_has_ip_enabled(args.user_smt_ip):
+        error_message = (
+            'Connection error: Could not establish a connection to {ip}.'
+            'Please, make sure the instance has that IP format enabled '
```

Comment:
s/the instance has that IP format enabled and working properly./the network configuration supports the provided IP address version./